### PR TITLE
syncer(dm): Improve logging of ignored TableMap/FormatDesc

### LINKED ISSUE #10204: Logging of FormatDesc and TableMap doesn't need to be a warning
### Is your feature request related to a problem?

DM logs a `unhandled event` for events it doesn't recognize. This includes TableMap and FormatDescriptionEvent. 

### Describe the feature you'd like

Lower log level for these expected events.

### Describe alternatives you've considered

_No response_

### Teachability, Documentation, Adoption, Migration Strategy

_No response_

### PR DESCRIPTION
**Pull request by dveeden**
### What problem does this PR solve?

Issue Number: close #10204 

### What is changed and how it works?

This makes the logging on the warning level less noisy and adds detail to the TableMap event logs.

#### Tests 

 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
Logging of `TableMapEvent` and `FormatDescriptionEvent` was changed from the warning level to the info level.
```

### CODE REVIEW

File: dm/syncer/syncer.go [hunk 1]
```diff
@@ -2442,6 +2442,13 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 			if needContinue {
 				continue
 			}
+		case *replication.TableMapEvent:
+			s.tctx.L().Info("unhandled TableMap event",
+				zap.String("schema", string(ev.Schema)),
+				zap.String("table", string(ev.Table)),
+			)
```
Comment (lance6716) [inline dm/syncer/syncer.go:2449]:
```suggestion
```

The "unhandled event" is used to detect some new events of future MySQL/MariaDB versions (I guess). So for these old type of events we don't need to log them, they are handled as no-op.

BTW we may also need to change line 2438

Comment (dveeden) [reply dm/syncer/syncer.go:2456]: @lance6716 The suggestion in your comment doesn't seem to change anything. Did you intent to suggest a code change?

I made changes based on your suggestion in ecc9985a47d63ed82d4bd059af4908ca9ba122ca

Comment (lance6716) [reply dm/syncer/syncer.go:2456]: I mean we don't need to print log for these two types of events. Just move them out of `default` case

Comment (dveeden) [reply dm/syncer/syncer.go]: Done in 5773c89d79b2ee8598414018ce1a11f975012bb0

**Review by okJiang — APPROVED** [review]

**Pull request merged**